server/chat-plugins/scavengers.ts

```diff
@@ -763,54 +907,74 @@ class ScavengerHunt extends Rooms.RoomGame {
 		});
 
 		if (filtered) return "Please do not leak the answer. Use /scavenge [guess] to submit your guess instead.";
```

Comment:
I think we should have a `/guess` function which aliases to `/choose`, for this.

Reply:
traditionally scavengers has always used /scavenge [answer] for guessing before the advent of the roomgame system, which is why we're keeping the same commands.